### PR TITLE
DisCatSharp .NET7

### PR DESCRIPTION
**Pull request by NyuwBot**
Closes T11

### CODE REVIEW

**Comment by Lulalaby** [comment]
To keep track of the stuff

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -2,16 +2,19 @@
...
+    "ghcr.io/devcontainers/features/git-lfs:1": {
+      "version": "latest"
     },
-	"ghcr.io/devcontainers/features/sshd:1": {
-        "version": "latest"
+    "ghcr.io/devcontainers/features/sshd:1": {
+      "version": "latest"
+    }, 
+    "ghcr.io/devcontainers/features/dotnet:1": {
+      "version": "7.0.100"
```
Comment (Lulalaby) [inline .devcontainer/devcontainer.json:17]: Need to figure that out

Comment (JBraunsmaJr) [reply .devcontainer/devcontainer.json:17]: Ah so that's why the global.json said 7.0.100

**Comment by Lulalaby** [comment]
T11

**Pull request closed without merging**